data/sql/updates/pending_db_world/rev_1707833862449013500.sql

```diff
@@ -0,0 +1,8 @@
+-- disables buff stacking for items 49856, 49857, 49858, 49859, 49861, 49860
+INSERT INTO `spell_group` (`id`, `spell_id`, `special_flag`) VALUES
+(2, 70233, 16),
+(2, 70234, 16),
+(2, 70235, 16),
+(2, 70242, 16),
+(2, 70243, 16),
+(2, 70244, 16);
```

Comment:
I think it's fine to use id 1 instead of 2. but according to the wiki you should use a lowest available id above 1000 if the spell won't be used in core code